docs/projected-volume.md

```diff
@@ -0,0 +1,27 @@
+# Projected Volumes and Windows Pods
+
+A [projected volume](https://kubernetes.io/docs/concepts/storage/volumes/#projected) allows mapping of several existing
+volume sources into the same directory in the container. With the [proposal for file permission handling in projected
+service account volume](https://github.com/kubernetes/enhancements/pull/1598) enhancement, the projected files have
+the correct permissions set including container user ownership for Linux pods when `RunAsUser` is set in the Pod 
+`SecurityContext`. However, for Windows pods this is not the case when `RunAsUsername` is set, due to differences in 
+the way user accounts are managed in Windows. Windows stores and manages local user and group accounts in a database
+file called Security Account Manager (SAM). This database is not shared between the Windows container host and the
+running containers. This prevents the kubelet from setting correct ownership on the files in the projected volume.
+This problem is tracked by [Windows Pod with RunAsUserName and a Projected Volume does not honor file
+permissions in the volume](https://github.com/kubernetes/kubernetes/issues/102849) issue.
+
+##Bound Service Account Token Volume
+The [Bound Service Account Token Volume](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-token-volume)
+feature results in the ServiceAccount admission controller adding a projected volume instead of a Secret-based volume
+for the non-expiring service account token created by the Token Controller. This implies that on all Kubernetes 1.22+ 
+clusters, a projected volume with service account token will be present on all Pods. Now for Windows Pods`RunAsUserName`
+is set in the Pod `SecurityContext`, the ownership will not be set correctly. This problem can get exacerbated when 
```

Comment:
who will the owner be?

Reply:
>Will the container still be able to read the credentials from the bound service account token?

Yes the container will have read access to the credentials.